src/main/kotlin/uk/gov/justice/digital/hmpps/prisonercontactregistry/client/PrisonApiClient.kt

```diff
@@ -19,17 +23,37 @@ class PrisonApiClient(
   val apiTimeout: Duration,
 ) {
 
+  companion object {
+    val logger: Logger = LoggerFactory.getLogger(this::class.java)
+  }
+
   private val contacts = object : ParameterizedTypeReference<ContactsDto>() {}
   private val addresses = object : ParameterizedTypeReference<List<AddressDto>>() {}
 
-  fun getOffenderContacts(offenderNo: String): ContactsDto? {
+  fun getOffenderContacts(offenderNo: String, approvedVisitorsOnly: Boolean): ContactsDto? {
```

Comment:
instead of a null field, could this have defaulted to false? If it's false, you could append it to the URL regardless?